src/pages/trade-options/option-tabs/option-tabs.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../variables";
+
```

Comment:
How would this look like if we can't fit all dates into the tabs?

Reply:
I was watching, bit.com, they don't care about responsive design, they are simply ignoring mobile devices and showing all same layout like it's for desktop. I think we should go the same way especially because we are changing the layout a lot and I think that we are not sure yet how our final design will look like. 
What kind of approach would you take?

Reply:
ACK